libs/auth-api-lib/src/lib/delegations/delegation-scope.service.ts

```diff
@@ -161,6 +167,47 @@ export class DelegationScopeService {
     })
   }
 
+  private findValidGeneralMandateTo(
+    toNationalId: string,
+    fromNationalId: string,
+  ): Promise<Delegation[]> {
+    const today = startOfDay(new Date())
+
+    return this.delegationModel.findAll({
+      where: {
+        toNationalId: toNationalId,
+        fromNationalId: fromNationalId,
+      },
+      include: [
+        {
+          model: DelegationDelegationType,
+          where: {
+            validTo: { [Op.or]: [{ [Op.is]: undefined }, { [Op.gte]: today }] },
+          },
+          required: true,
+          include: [
+            {
+              model: DelegationTypeModel,
+              required: true,
+              where: {
+                id: AuthDelegationType.GeneralMandate,
+              },
+              include: [
+                {
+                  model: ApiScope,
+                  required: true,
+                  where: {
+                    enabled: true,
+                  },
+                },
+              ],
+            },
+          ],
+        },
+      ],
+    })
```

Comment:
I think its better to split this into:

1. Query `DelegationModel` to check if a valid GeneralMandate delegation is found. If not found return empty array
2. If valid then find all api scopes that has GeneralMandate relation.